plugins/catalog-backend-module-github/src/lib/github.ts

```diff
@@ -214,7 +230,7 @@ export async function getOrganizationUsers(
     query,
     org,
     r => r.organization?.membersWithRole,
```

Comment:
I think we should do the filtering here.

```suggestion
    r => r.organization?.membersWithRole.filter(u => excludeSuspendedUsers ? !u.suspendedAt : true),
```